packages/components/package.json

```diff
@@ -22,7 +22,7 @@
         "polished": "^4.2.2",
         "react-date-range": "^1.4.0",
         "react-is": "16.13.1",
-        "react-select": "^3.1.1",
+        "react-select": "^4.3.1",
```

Comment:
You locked version in `packages/suite`. Unify it.

Reply:
f99eb9bff7216416f069322f34e684b2b7e7dbf4